arch/arm/src/s32k3xx/s32k3xx_edma.c

```diff
@@ -1623,4 +1659,4 @@ void s32k3xx_dmadump(const struct s32k3xx_dmaregs_s *regs, const char *msg)
   dmainfo("      DMAMUX: %08x\n", (unsigned int)regs->dmamux);
 }
 #endif /* CONFIG_DEBUG_DMA */
-#endif /* CONFIG_S32K3XX_EDMA */
+#endif /* CONFIG_S32K3XX_EDMA */
```

Comment:
why all header file remove the last \n?

Reply:
Oops something went wrong there.
There were some double \n\n which I wanted to remove but apparently I've removed them all.
I've pushed the correct version now.